regression/valgrind_suppress.txt

```diff
@@ -893,14 +893,23 @@
    openmpi210/e001b
```

Comment:
Is this suppression label supposed to be the same as at line 884? (Apologies if I'm missing something.)